vulture_os/services/templates/services/frontend_edit.html

```diff
@@ -464,7 +464,7 @@ <h4 class="panel-title"><i class="icon fa fa-ban"></i> {% translate "Form errors
                           </div>
                         </div>
                       </div>
-                      <div class="row log-mode haproxy-conf">
+                      <div class="row log-mode">
```

Comment:
Unfortunately, the parameter will appear for every type of log frontend.
Even thoses who doesn't have haproxy configuration.